src/main/java/com/opentable/callback/ExecutorBatchingCallback.java

```diff
@@ -93,8 +93,8 @@ public void call(final List<T> item) throws Exception
             inFlight.incrementAndGet();
             executor.submit(new ExecutorCallable<T>(out, item));
 
-            Future<Void> f;
-            while ( (f = executor.poll()) != null ) {
+            Future<Void> f = executor.poll();
+            while (f != null) {
```

Comment:
Open to other suggestions on how to unwrap the assignment-in-loop-condition. (Can also just suppress it out)